MMUL-BASH-01/Exercises/Day-4/Exercise.003

```diff
@@ -0,0 +1,8 @@
+# Exercise 003 - Guess the passphrase!
+
+1) The 'read' command can be used to read the user input and save in a variable.
+  The command
```

Comment:
Wrong indentation: you need to be under the first "The" for the following lines.